README.md

```diff
@@ -13,8 +13,8 @@ To download and setup the software:
 ~~~~
 git clone https://github.com/naterush/version-control.git;
 cd version-control;
-virtualenv venv;
-source venv/bin/activate;
+python3 -m venv ~/.virtualenvs/myvenv;
```

Comment:
I don't think this is standard usage. We probably just wanna use the myvenv command raw.

Reply:
Ok. It is how I was able to get my virtual environment to run Python 3 because I think it defaults to Python 2, but perhaps it has something to do with my specific configuration.